specification/index.md

```diff
@@ -1,48 +1,42 @@
-		- [5.1. General requirements for metadata in a CSIP Information Package](#51-general-requirements-for-metadata-in-a-csip-information-package)
-		- [5.2. General requirements for the use of metadata](#52-general-requirements-for-the-use-of-metadata)
-		- [5.3. Use of METS](#53-use-of-mets)
-	    - [5.3.1.	Use of the METS root element (element mets)](#531-use-of-the-mets-root-element-element-mets)
-	    - [5.3.2.	Use of the METS header (element metsHdr)](#532-use-of-the-mets-header-element-metshdr)
-	    - [5.3.3 Use of the METS descriptive metadata section (element dmdSec)](#533-use-of-the-mets-descriptive-metadata-section-element-dmdsec)
-	    - [5.3.4.	Use of the METS administrative metadata section (element amdSec)](#534-use-of-the-mets-administrative-metadata-section-element-amdsec)
-	    - [5.3.5.	Use of the METS file section (element fileSec)](#535-use-of-the-mets-file-section-element-filesec)
-	    - [5.3.6.	Use of the METS structural map (element structMap)](#536-use-of-the-mets-structural-map-structmap)
-		- [5.4. Use of PREMIS](#54-use-of-premis)
-	- [6. Implementation considerations](#6-implementation-considerations)
-	  - [6.1.	Content Information Type Specifications](#61-content-information-type-specifications)
-	    - [6.1.1.	What is a Content Information Type Specification?](#611-what-is-a-content-information-type-specification)
-	    - [6.1.2.	Maintaining Content Information Type Specifications](#612-maintaining-content-information-type-specifications)
-	  - [6.2. Handling large packages](#62-handling-large-packages)
-	  - [6.3.	Handling descriptive metadata within the Common Specification](#63-handling-descriptive-metadata-within-the-common-specification)
+	- [SIP structure](#csip-structure)
+	  - [older structure of the CSIP](#folder-structure-of-the-csip)
+	  - [mplementing the structure](#implementing-the-structure)
+	- [se of metadata](#use-of-metadata)
+		- [General requirements for metadata in a CSIP Information Package](#general-requirements-for-metadata-in-a-csip-information-package)
+		- [General requirements for the use of metadata](#general-requirements-for-the-use-of-metadata)
+		- [Use of METS](#use-of-mets)
+	    - [	Use of the METS root element (element mets)](#use-of-the-mets-root-element-element-mets)
+	    - [	Use of the METS header (element metsHdr)](#use-of-the-mets-header-element-metshdr)
+	    - [Use of the METS descriptive metadata section (element dmdSec)](#use-of-the-mets-descriptive-metadata-section-element-dmdsec)
+	    - [	Use of the METS administrative metadata section (element amdSec)](#use-of-the-mets-administrative-metadata-section-element-amdsec)
+	    - [	Use of the METS file section (element fileSec)](#use-of-the-mets-file-section-element-filesec)
+	    - [	Use of the METS structural map (element structMap)](#use-of-the-mets-structural-map-structmap)
+		- [Use of PREMIS](#use-of-premis)
+	- [mplementation considerations](#implementation-considerations)
```

Comment:
Are we missing letters?

Reply:
This file's to go so don't worry about it.